### PR TITLE
chore: add Cluster Name title and change helm value

### PR DESCRIPTION
**Pull request by amandavialva01**
## Description
- Add "Cluster Name" title as shown below
-  change helm chart `additional_resource_managers[i].name` to `additional_resource_managers[i].cluster_name`

Before (No title)
NoClusterNameTitle

After (with title)
 
WithClusterNameTitle

## Test Plan
CI Passes.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/components/WorkspaceCreateModal.tsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
             <>
               {resourceManagers.map((name) => (
                 <Fragment key={name}>
-                  <Form.Item label={name} name={['bindings', name, 'namespace']}>
+                  <Form.Item label={'Cluster Name: ' + name} name={['bindings', name, 'namespace']}>
```
Comment (keita-determined) [inline webui/react/src/components/WorkspaceCreateModal.tsx:201]: can we use template literal?

Comment (amandavialva01) [reply webui/react/src/components/WorkspaceCreateModal.tsx]: Done!

File: helm/charts/determined/values.yaml [hunk 2]
```diff
@@ -363,7 +363,7 @@ resourcePools:
 #   - resource_manager:
 #       type: kubernetes
 #       max_slots_per_pod: 1
-#       name: carolina-multirm-1
+#       cluster_name: carolina-multirm-1
```
Comment (keita-determined) [inline helm/charts/determined/values.yaml:366]: its not from this PR, but i think `carolina-multirm-1` shouldn't be in the main branch?

**Review by keita-determined — APPROVED** [review]

File: helm/charts/determined/values.yaml (hunk 2 above)
Comment (amandavialva01) [reply helm/charts/determined/values.yaml]: Agreed! Changed this

**Review by kkunapuli — APPROVED** [review]

**Review by carolinaecalderon — APPROVED** [review]
LGTM, thanks for adjusting

**Pull request merged**